manifest.json

```diff
@@ -17,6 +17,13 @@
         "1.0.2"
       ]
     },
+    "codezero": {
```

Comment:
This needs to match the pkg folder name

Should be `codezero-rancher-extension`